nipype/pipeline/engine/tests/test_engine.py

```diff
@@ -626,6 +626,33 @@ def func1(in1):
     assert not error_raised
 
 
+def test_parameterize_dirs_false(tmpdir):
+    from ....interfaces.utility import IdentityInterface
+    from ....testing import example_data
+
+    input_file = example_data('fsl_motion_outliers_fd.txt')
+
+    n1 = pe.Node(EngineTestInterface(), name='Node1')
+    n1.iterables = ('input_file', (input_file, input_file))
+    n1.interface.inputs.input1 = 1
+
+    n2 = pe.Node(IdentityInterface(fields='in1'), name='Node2')
+
+    wf = pe.Workflow(name='Test')
+    wf.base_dir = str(tmpdir)
+    wf.config['execution']['parameterize_dirs'] = False
+    wf.connect([(n1, n2, [('output1', 'in1')])])
+
+    error_raised = False
+    try:
+        wf.run()
+    except Exception as ex:
```

Comment:
@satra : I don't understand this try/except block, since we expect `wr.run()` to work, and if we get ANY exception,  the test should fail (and pytest will inform what exception has been raised). IMO, we should remove try/except block at all, just keep `wr.run()`. The same is true about all test that have `assert not error_raised`.  And the test that have `assert error_raised` can be rewritten using ` with pytest.raises(Exception)`.

If you agree with me, I can clean whole file after this PR is merged (unless you prefer to do it before).